### PR TITLE
sra-tools: solves hanging behaviour for missing *_1 files for PE data

### PR DESCRIPTION
**Pull request by yhoogstrate**
I think this solves #1081. It does not give an error if something like this happens. Do we want this or would it be desired to return an error in the end as well?

### CODE REVIEW

**Comment by bgruening** [comment]
ping @galaxyproject/iuc

**Comment by mvdbeek** [comment]
I would prefer to error out, or to have an option like `fail when forward read of paired collection not present`. For larger scale analysis I think it's easier to react to failed datasets than to datasets that seem OK. But anything is better than it is now, so 👍

File: tools/sra-tools/fastq_dump.xml [hunk 1]
```diff
@@ -94,21 +94,37 @@
 
 
     #if str( $outputformat ) == "fasta":
-
-        for f in *_2.fasta ; do   mv "\$f" "`basename \$f _2.fasta`_reverse.fasta" ;  mv "`basename \$f _2.fasta`_1.fasta" "`basename \$f _2.fasta`_forward.fasta"  ; done &&
-        for f in *_1.fasta; do mv "\$f" "`basename \$f _1.fasta`__single.fasta"; done
-
+        for f in *_2.fasta ; do
+            basename=`basename \$f _2.fasta` &&
```
Comment (nsoranzo) [inline tools/sra-tools/fastq_dump.xml]: Should `\$f` be double-quoted?

File: tools/sra-tools/fastq_dump.xml [hunk 2]
```diff
@@ -94,21 +94,37 @@
...
+            mv "\$f" "\$basename"_reverse.fasta &&
+            
+            ## in exceptional cases this file may be absent
+            if [ ! -e "\$basename"_1.fasta ];
+            then
+               touch "\$basename"_1.fasta &&
+            fi ;
+            
+            mv "\$basename"_1.fasta "\$basename"_forward.fasta &&
+          done &&
```
Comment (nsoranzo) [inline tools/sra-tools/fastq_dump.xml]: Indentation

**Pull request closed without merging**